spk-config.yaml

```diff
@@ -23,6 +23,7 @@ introspection:
     table_name: "storage-account-table-name" # Must be defined to run spk deployment commands
     partition_key: "storage-account-table-partition-key" # Must be defined to run spk deployment commands
     key: "storage-access-key" # Must be defined to run spk deployment commands. Use ${env:INTROSPECTION_STORAGE_ACCESS_KEY} and set it in .env file
+    source_repo_access_token: "source_repo_access_token" # Optional. Define this to see author names in the dashboard
```

Comment:
Is it different from `azure_devops.access_token? if it is, the source repo can belong to some other AzDo project.

Reply:
@sarath-p Yes it can be different if the src repo is on GitHub and `azure_devops.access_token` is for the pipeline access token. Ran into this scenario yesterday with Tim's setup and he needed two separate tokens.

Reply:
Do you expect the source repo access token work for hld and manifests repos too?  I am not sure if it requires the author name from hld and manifests repos as it progresses in the flow.

Reply:
@sarath-p We don't need the author information from HLD and manifests because only automated commits would go into it (ideally). So just a source repo access token is all we need